css/evaluations.css

```diff
@@ -0,0 +1,110 @@
+body{
+	background-color: white;
+}
+
+.logoClass{
+	background-image: url('https://www.softvision.ro/images/common/logo-v2.png');
+	width: 200px;
+	height: 37.40px;
+	background-size: contain;
+	margin-left: 50px;
+	margin-top: 20px;
+	float: left;
+	margin-right: 150px;
+}
+
+header{
+	width: 100%;
+	height: 80px;
+	box-shadow: 0px 3px 10px grey;
+}
+
+footer{
+	margin-bottom: 0;
+}
+
+header span{
+	float: right;
+}
+
+.formated{
+	color: black;
+	font-size: 22px;
+}
+
+nav span{
+	margin-top: 30px;
+}
+
+span a{
+	color: black;
+	text-decoration: none;
+	margin-right: 60px;
+}
+
+.buttoni{
+	width: 250px;
+	height: 70px;
+	float: left;
+	text-align: center;
+	line-height: 70px;
+	cursor: pointer;
+}
+
+.buttoni:hover{
+	cursor: pointer;
+}
+
+.selected{
+	border-bottom: 10px solid #1ab2ff;
+}
+
+.mainTableClass{
+	margin: 0 auto;
+	margin-top: 100px;
+	font-size: 20px;
+	border-collapse: collapse;
+	margin-bottom: 100px;
+	width: 60%;
+}
+
+.mainTableClass th{
+	text-align: left;
+}
+
+.firstRow{
+	background-color: #1ab2ff;
+	color: white;
+	height: 60px;
+	border: 1px solid #1ab2ff
+}
+
+.firstRowEvaluations{
```

Comment:
What if I told you there are ways of targeting elements with particular indices in the DOM without adding extra classes )(e.g., `firstRow` / `firstRowEvaluations`) to them? Would you be able to tell me what alternatives are available?